flutter-for-ios.md

```diff
@@ -426,17 +423,22 @@ class SignatureState extends State<Signature> {
 }
 {% endprettify %}
 
+## Where is the widget's opacity?
+
+On iOS, everything has .opacity or .alpha. In Flutter, most of the time you need to
```

Comment:
It's good but I think it could use an explanation why. It seems like extra composing. Maybe there's a benefit?

Reply:
I've asked Ian, and he wasn't sure. I'm going to merge this, for now.